Resources/Prototypes/_Scp/Actions/scp106_actions.yml

```
@@ -59,4 +80,39 @@
     raiseOnUser: true
     itemIconStyle: BigAction
     useDelay: 60
-    # TODO: ICON
+    icon:
+      sprite: _Scp/Actions/scp106-actions.rsi
+      state: scp106_phantom
+
+- type: entity
+  id: Scp106PassThrough
+  name: ХЗ.
+  description: "Позволяет пройти любое препятствие"
+  components:
+  - type: InstantAction
+    event: !type:Scp106PassThroughAction
+    raiseOnUser: true
+    itemIconStyle: BigAction
+    useDelay: 15
+    icon:
+      sprite: _Scp/Actions/scp106-actions.rsi
+      state: scp106_phantom
+
+### PHANTOM ABILITIES END
+
+### 3 TIRES OF PHANTOM ABILITY START
+
+- type: entity
+  id: Scp106BecomePhantom
+  name: Стать фантомом.
+  description: "Новое воплощение для поиска трупов. Фантом восстанавливается каждые 5 минут."
```

```suggestion
  description: Новое воплощение для поиска трупов. Фантом восстанавливается каждые 5 минут.
```

В прототипах не нужно помечать строки через "", это только засоряет прототип